collections/_article/submissions-open-godot-2024-showreel.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Submissions open for Godot 2024 showreel!"
+excerpt: "Showcase your Godot creations in 2024! We're inviting submissions of short videos featuring your published or in-progress Godot games or tools. Don't miss this chance to be a part of it!"
+categories: ["news"]
+author: Emi
+image: /storage/blog/covers/godot-showreel-2024.webp
+image_caption_description: Illustration by André Marí Coppola
+date: 2024-09-05 15:00:00
+---
+
+Showcase your Godot Engine 2024 creations! We invite you to submit videos featuring your published or in-progress games or tools so we can make the Godot 2024 Showreel.
+
+The showreels are an informative and celebratory showcase, to highlight the wide array of projects developed using the Godot Engine. If you want to participate and share your Godot-made game, app or tool submit your entry starting today.
+
```

Comment:
```suggestion

## Submission requirements:

When submitting your video, please make sure to ensure that these requirements are met. We withold the right to disqualify submissions that do not follow the rules.

- Your game or tool has to be made in Godot
- If your game was featured in last year's showreel, it can not be submitted again
- Submit your game to one catergory only; the final showreel combines all entries into one video
- Your video should be short, ideally between 1 and 3 minutes.
- Do not add logos or text overlays to your video. Our final edit will incorporate watermarks to credit you
- Your video must be captured with good quality and stable framerate. If your project uses Godot 4, consider using [Movie Maker mode](https://docs.godotengine.org/en/latest/tutorials/animation/creating_movies.html)
- All content has to be SFW. Avoid nudity, gore, hateful or shocking content

For desktop games, we recommend the minimum resolution to be 1080p. 60 FPS is a big plus. For mobile games, 720p or higher. The final showreel video will be edited and uploaded in 4K @ 60 FPS to maximize quality.

Don't worry about not having a perfect video! If your game is selected, we may contact you to discuss the video content and give you a chance to improve your submission for the final edit.

```